From 178c562a4621162dbe19a7c34fa2ad558684f40e Mon Sep 17 00:00:00 2001
From: "Dr. Stephen Henson" <steve@openssl.org>
Date: Tue, 6 Jan 2015 20:55:38 +0000
Subject: [PATCH] use correct function name

(Upstream commit 178c562a4621162dbe19a7c34fa2ad558684f40e OpenSSL_1_0_1-stable)

Upstream-Status: Backport

Reviewed-by: Rich Salz <rsalz@openssl.org>
Reviewed-by: Matt Caswell <matt@openssl.org>
Signed-off-by: Yue Tao <yue.tao@windriver.com>
(cherry picked from commit cb62ab4b17818fe66d2fed0a7fe71969131c811b)
---
 crypto/asn1/a_verify.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/asn1/a_verify.c b/crypto/asn1/a_verify.c
index a571009..78dde1d 100644
--- a/crypto/asn1/a_verify.c
+++ b/crypto/asn1/a_verify.c
@@ -154,7 +154,7 @@ int ASN1_item_verify(const ASN1_ITEM *it, X509_ALGOR *a,
 
 	if (signature->type == V_ASN1_BIT_STRING && signature->flags & 0x7)
 		{
-		ASN1err(ASN1_F_ASN1_VERIFY, ASN1_R_INVALID_BIT_STRING_BITS_LEFT);
+		ASN1err(ASN1_F_ASN1_ITEM_VERIFY, ASN1_R_INVALID_BIT_STRING_BITS_LEFT);
 		return -1;
 		}
 
-- 
1.7.9.5

